scripts/ts-api-extractor/.env.example

```
@@ -0,0 +1,15 @@
+DEEPL_API_KEY=your-deepl-api-key-here
+# Optional: override default endpoint (https://api-free.deepl.com/v2/translate)
+# Use https://api.deepl.com/v2/translate for Pro plan
+DEEPL_ENDPOINT=
+
+# Optional: pre-registered DeepL Glossary ID for domain term locking
+# Create via DeepL API and paste the ID here
+DEEPL_GLOSSARY_ID=
+
+# LiteLLM Proxy (required for LLM MQM evaluation and post-editing)
+LITELLM_BASE_URL=
+LITELLM_API_KEY=your-litellm-api-key-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LITELLM_API_KEY key should go before the LITELLM_BASE_URL key

(UnorderedKey)